client/python/samples/resnet_grpc_predict.py

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import numpy as np
+import classes
+from ovmsclient import make_grpc_client, make_grpc_predict_request
+
+parser = argparse.ArgumentParser(description='Make prediction using images in numpy format')
+parser.add_argument('--images_numpy_path', required=False, default='./utils/imgs.npy',
+                    help='path to image')
+parser.add_argument('--labels_numpy_path', required=False,
+                    help='numpy in shape [n,1] - can be used to check model accuracy')
+parser.add_argument('--grpc_address', required=False, default='localhost',
+                    help='Specify url to grpc service. default:localhost')
+parser.add_argument('--grpc_port', required=False, default=9000,
+                    help='Specify port to grpc service. default: 9000')
+parser.add_argument('--model_name', default='resnet', help='Model name to query. default: resnet',
+                    dest='model_name')
+parser.add_argument('--model_version', default=0, type=int,
+                    help='Model version to query. Lists all versions if omitted',
+                    dest='model_version')
+parser.add_argument('--input_name', required=False, default='0',
+                    help='Specify input tensor name. default: 0')
+parser.add_argument('--output_name', required=False, default='1463',
+                    help='Specify output name. default: 1463')
+parser.add_argument('--batchsize', default=1, type=int,
```

Comment:
Let's drop that parameter. Minimal amount of options would be clearer to the user.